versions/3.1.0.md

```diff
@@ -75,6 +75,8 @@ Path templating refers to the usage of template expressions, delimited by curly
 
 Each template expression in the path MUST correspond to a path parameter that is included in the [Path Item](#path-item-object) itself and/or in each of the Path Item's [Operations](#operation-object).
 
+The value for these path parameters MUST NOT contain any unescaped "generic syntax" characters described by [RFC3986](https://tools.ietf.org/html/rfc3986#section-3): forward slashes (`/`), forward question marks (`?`), or hashes (`#`).
```

Comment:
Consider rephrasing as an instruction for how to encode these special characters:

> The following characters MUST be percent-encoded when used in the value of a path parameter: forward slash (`/`), question mark (`?`), number sign (`#`).

(or similar)

Note that [RFC 3986](https://tools.ietf.org/html/rfc3986#section-1.2.3) refers to `#` as the "number sign", I'd suggest using the same term for consistency.